mergebot-config.json

```diff
@@ -1,11 +1,19 @@
 {
+   "backport": true,
+   "backport-strategy": {
+       "interactive": false,
+       "strategy": "single-repo",
+       "copy-labels": true
+   },
    "update_jira_on_merge": {
      "enabled": true,
      "interactive": false,
      "fix_version_map": {
-       "master": "K8s Base Addons 3.0.0",
+       "master": "K8s Base Addons Next",
```

Comment:
I think this should be `K8s Base Addons 3.3.0`

If you put Next, that's what will be put in JIRA whenever you close a PR and then you'll have to go into JIRA and re-update those tickets to point to the correct KBA version.

If you miss updating in JIRA and start merging fixes for the next release, then you'll have 2 versions you need to figure out in JIRA. This will become a nightmare for you (or whoever shares the pain).

Reply:
I chose "next" because we won't know in master if there's going to be a breaking change before the next release that will cause us to use a different major version until we cut the soak branch. Any ideas how that should be handled?

Reply:
How about two different branches?

One for `main` and one for `soak`. Then you know which version soak is pointing to and what fixes need to go in there. The rest can be merged to main and then you update the versions in mergebot-config when the next soak is cut. Or am I misunderstanding your question?